index.js

```diff
@@ -0,0 +1,85 @@
+const express = require('express');
+const db = require('./data/db.js');
+const bodyParser = require('body-parser');
+
+const server = express();
+server.use(bodyParser.urlencoded({ extended: true }));
```

Comment:
Express now includes body-parser middleware by default! So just use `server.use(express.json());` in lieu of bodyParser.